<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>-1</y>
      <width>781</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Intelligent Banking Management System (IBMS)</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="menuBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>â‰¡</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="sideMenu">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>69</y>
      <width>231</width>
      <height>491</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>231</width>
      <height>0</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Transaction &amp; Account Management</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Transaction Classification</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Customer Queue &amp; Event Simulation </string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>278</width>
      <height>467</height>
     </rect>
    </property>
    <widget class="QWidget" name="pageM1">
     <widget class="QLineEdit" name="m1InputBar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>ID, Name, Balance</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboM1Part">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>411</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnM1Enter">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="m1Output">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>451</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageM3">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="comboM3Part"/>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="m3InputBar"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnM3Enter">
        <property name="text">
         <string>Enter Queue</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QTextEdit" name="m3Output"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="pageM2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="comboM2Part"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelCategories">
        <property name="text">
         <string>Categories
0 - Food
1 - Utilities
 2 - Education
 3 - Government
 4 - Others
</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="inputDescription">
        <property name="placeholderText">
         <string>TxID, Description (e.g., 1, Pizza)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btnClassify">
        <property name="text">
         <string>Enter</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QTextEdit" name="lblResult">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
